### PR TITLE
Remove Aurelia aliases

### PR DESCRIPTION
**Pull request by Nixinova**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)

----

Removes all aliases of Aurelia.
Most are completely unused and others (like JSX) are not related to this.

### CODE REVIEW

File: topics/aurelia/index.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
 ---
-aliases: aureliajs, aurelia-vnext, vnext, aurelia-spa, spa, jsx, aureliavnext, aurelia-2, aurelia-2-vnext
+topic: aurelia
```
Comment (itsbagpack) [inline topics/aurelia/index.md:2]: Could we keep this alphabetized and back on L10?

**Comment by Vannsdok** [comment]
Nixinova:patch-1

**Comment by Vannsdok** [comment]
Nixinova:patch-1

**Comment by Nixinova** [comment]
Why does this repo attract so much spam?...

**Review by Gulierenee — COMMENTED** [review]
My Pages app/ want to read it, needs to be put in English

**Review by Gulierenee — COMMENTED** [review]
Excited to be here!!!!

**Review by kenyonj — APPROVED** [review]
Thank you!

**Pull request merged**